docs/recipes/react.md

```diff
@@ -0,0 +1,69 @@
+# React testing
+
+With 0.10.0, AVA stopped including your `.babelrc` (for reasons outlined in [this PR](https://github.com/sindresorhus/ava/pull/398)), which means you can't use JSX in your test/spec files. Note that this doesn't apply to your source files, they can still be written in JSX!
```

Comment:
I was under the impression that there is soon to be a release that allows you to specify a `.babelrc` to use or some way to configure babel yourself. Is this not true?